styleguide/source/_patterns/01-molecules/08-lists/11-list-events/_list-events.scss

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file list-events.scss
+ *
+ */
+
+.list-events {
+  margin: 0;
+  padding: 0;
+}
+
+.list-events_item {
+  border-top: 1px solid $gray-8;
+  list-style-type: none;
+  margin-bottom: 0;
+  padding: 30px 0;
+  @include breakpoint($bp-small) {
+    padding: 40px 0;
+  }
+}
+
+.list-events_item_summary,
+.list-events_item_date {
+  padding-bottom: 9px;
+}
+
+.list-events_item_summary,
+.list-events_item_date,
+.list-events_item_location {
+  @include grid__unit--cols(12);
+  display: block;
+  vertical-align: text-top;
+}
+
+.list-events_item p {
+  margin-bottom: 0;
+}
+
+.list-events_item_summary a {
+  display: inline-block;
+  font-size: 24px;
+  padding-bottom: 9px;
+
+  .icon-external {
+    height: 9px;
+    width: 9px;
+    margin-left: 7px;
+  }
+
+  &:hover .external {
+    fill: $blue; 
+  }
+}
+
+.list-events_item_summary_title-sold-out:after {
+  content: "Sold Out";
```

Comment:
Suggestion to add comment that 'sold out' will require some logic, and that the content should be added dynamically and in a way so that it is translatable.